mods/ra/chrome/ingame-observer.yaml

```diff
@@ -412,6 +412,14 @@ Container@OBSERVER_WIDGETS:
 									Text: Harvesters
 									Align: Right
 									Shadow: True
+								Label@DERRICKS_HEADER:
```

Comment:
This makes the panel too wide at our minimum horizontal resolution (1024px):

<img width="1024" alt="Screenshot 2019-09-06 at 17 10 02" src="https://user-images.githubusercontent.com/167819/64443202-58e64280-d0c9-11e9-836c-4002598ccc5b.png">

The "Earned this min" "Harvesters" and "Oil Derricks" columns are clearly far too wide for their data, but I don't have a simple solution to reduce their headers...

Reply:
"Harvesters" and "Oil Derricks" could be replaced by pictograms. That's not a short term solution though, unless someone already has such pictograms lying around.

Reply:
Think they already have customized GPS dots, which can pass as such.